src/components/GameDataModal/GameDataModal.jsx

```diff
@@ -57,6 +57,14 @@ export default function GameDataModal({ isModalOpen, closeModal, openModal, setM
         setShowingDescription(false);
     }
 
+    if (isModalOpen) {
```

Comment:
It is not a good practice to manipulate DOM in react. Maybe is there another way to toggle that class when modal opens.

Reply:
Sadly, I don't think there's another way of updating the body classlist. I avoided doing that by using ref in most cases, but this is a special one.